src/erlfmt_format.erl

```diff
@@ -259,9 +259,24 @@ do_expr_to_algebra({clauses, _Meta, Clauses}) ->
     clauses_to_algebra(Clauses);
 do_expr_to_algebra({body, _Meta, Exprs}) ->
     block_to_algebra(Exprs);
-do_expr_to_algebra({sigil, _Meta, Prefix, Content, Suffix}) ->
+do_expr_to_algebra({sigil, _Meta, Prefix, {string, ContentMeta, _Value}, Suffix}) ->
     PrefixDoc = concat(<<"~">>, do_expr_to_algebra(Prefix)),
-    concat(concat(PrefixDoc, do_expr_to_algebra(Content)), do_expr_to_algebra(Suffix));
+    Text = erlfmt_scan:get_anno(text, ContentMeta),
+    SigilDoc = concat(concat(PrefixDoc, string(Text)), do_expr_to_algebra(Suffix)),
+    case string:split(Text, "\n", all) of
+        % Triple-quoted sigils don't need force_breaks
+        ["\"\"\"" ++ _ | _] -> SigilDoc;
```

Comment:
This will not work correctly for multiple nesting levels. We need to do the same logic as is done in normal triple-quoted strings in `string_to_algebra`, just add prefix and suffix around it.
Furthermore, ideally we'd avoid duplicating this logic and just share the code.